src/site/stages/build/drupal/page.js

```diff
@@ -120,19 +120,19 @@ function updateEntityUrlObj(page, drupalPagePath, title, pathSuffix) {
       .replace(/&/g, '')
       .replace(/\s+/g, '-')
       .toLowerCase();
+
   let generatedPage = Object.assign({}, page);
+  const absolutePath = path.join('/', drupalPagePath, pathSuffix);
+
   generatedPage.entityUrl.breadcrumb = [
     ...page.entityUrl.breadcrumb,
     {
-      url: { path: drupalPagePath },
+      url: { path: absolutePath },
       text: page.title,
     },
   ];
-  generatedPage = set(
-    'entityUrl.path',
-    `${drupalPagePath}/${pathSuffix}`,
-    page,
-  );
+
+  generatedPage = set('entityUrl.path', absolutePath, page);
```

Comment:
The Metalsmith file path should not lead with a `/`. If so, it'll try to write to the root of the file system. This didn't matter when we were manually adding `drupal` into the paths, but it does now that `drupal` is dynamic.